docs/t-sql/statements/create-index-transact-sql.md

```diff
@@ -138,7 +138,7 @@ CREATE [ UNIQUE ] [ CLUSTERED | NONCLUSTERED ] INDEX index_name
 }
 
 <filter_predicate> ::=
-    <conjunct> [ AND <conjunct> ]
+    <conjunct> [ AND <conjunct> ] [ ...n ]
```

Comment:
If a change must be done, the correct notation here is `<conjunct> [ AND ] [ ...n ]`

Reply:
Thanks for the feedback @pmasl. Change made accordingly.